pgm-2.txt

```diff
@@ -0,0 +1,7 @@
+
+#!/bin/bash
+echo "hellooooooo"
+find *.log > z.txt
+find *.txt >> z.txt
+sudo mkdir /home/sudhir/zout
+tar -zczf matches.tar.gz /home/main
```

Comment:
I have committed the new program with your review comments. Please review the new program and let me know the review comments